packages/nx/src/config/workspaces.ts

```diff
@@ -33,7 +33,10 @@ import { joinPathFragments } from '../utils/path';
 export function workspaceConfigName(
   root: string
 ): 'angular.json' | 'workspace.json' | null {
-  if (existsSync(path.join(root, 'angular.json'))) {
+  if (
+    existsSync(path.join(root, 'angular.json')) &&
+    existsSync(path.join(root, 'nx.json'))
```

Comment:
I am using "does nx.json exists" to answer the question "is Nx being directly used in the repo (and not via Lerna).